step-certificates/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: step-certificates
-version: 1.23.2
-appVersion: 0.23.2
+version: 1.23.3
+appVersion: 0.23.3
```

Comment:
`0.23.3` hasn't been released yet, so the appVersion should be `0.23.2`, to keep the version consistent with the `appVersion` we can also name it something like `1.23.2+1`

Reply:
```suggestion
version: 1.23.2+1
appVersion: 0.23.2
```